deploy/README.md

```diff
@@ -92,9 +92,6 @@ HOME_AMB_BRIDGE=0x
 # to the Home network.
 FOREIGN_AMB_BRIDGE=0x
 # The gas limit that will be used in the execution of the message passed to the mediator contract
-# in the Foreign network.
-HOME_MEDIATOR_REQUEST_GAS_LIMIT=2000000
```

Comment:
May you consider to add the forwarding rules manager deployment to be configured through the `.env` file as well? It cold be done in a separate PR.

Reply:
Sure, will do it in a separate PR.